ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -568,5 +568,4 @@ data TraceReplayEvent blk
   -- | Signals that we are about to compute and push a ledger state into the
   -- LedgerDB. Used while replaying the VolatileDB (during Initial Chain
   -- Selection).
-  | UpdateLedgerDbTraceEvent (UpdateLedgerDbTraceEvent blk)
```

Comment:
Dangling comment?

Reply:
wow, got so used to them being below the code, thx!